test.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash -ex
```

Comment:
Do we need to use `-x` here, very verbose? We tend to vomit everything into jenkins and then it's hard to really read the logs. This is a bigger issue than this PR, but I'm curious - do you think the api-ruby build is too noisy or is it fine?

https://jenkins.conjur.net/job/api-ruby-pipeline/job/tests_170626/8/consoleFull